suite-native/ethereum-tokens/src/ethereumTokensSelectors.ts

```diff
@@ -124,7 +124,7 @@ const selectAccountTransactionsWithTokensWithFiatRates = memoizeWithArgs(
             A.map(transaction => ({
                 ...transaction,
                 tokens: pipe(
-                    transaction.tokens,
+                    transaction?.tokens ?? [],
```

Comment:
This should be fixed in types too if it makes issues here, it can make issues in other places.

Reply:
Fixing this in types would also mean cascade update anywhere this is being used. I can create issue for this to handle later properly outside of this PR not to make this bigger. wdyt?

Reply:
That basically means that it can break app in any of these places. Please create issue for that and I would even say that this should have some kind of priotity.